.github/workflows/ohos.yml

```diff
@@ -151,12 +151,77 @@ jobs:
       dromaeo: false
     secrets: inherit
 
+  # Generate a unique id that allows the timeout
+  # jobs to find the workload job run (via the job’s friendly name), even
+  # if there are multiple instances in the workflow call tree.
+  gen-uuids:
+    name: Generate unique runner IDs
+    runs-on: ubuntu-latest
+    outputs:
+      # re-use the same uuid for build, test, bench, with a prefix
```

Comment:
Can you document here that this is safe because they run sequentially or even better separate them. While this works now, I suspect when somebody in the future edits this workflow (and makes stuff run in parallel) this will become a pitfall if not careful enough.

Reply:
They can run in parallel just fine, since we prefix the `uuid` with `bench`/`test`/`build`, so they are still unique, but slightly more human readable in the sense that one can see that these jobs belong to the same workflow instance.